.github/workflows/ci.yml

```diff
@@ -51,3 +51,8 @@ jobs:
                   HAZELCAST_ENTERPRISE_KEY: ${{ secrets.HAZELCAST_ENTERPRISE_KEY }}
               run: |
                   npm run coverage
+            - name: Publish to Codecov
+              if: ${{matrix.os == 'ubuntu-latest'}}
+              uses: codecov/codecov-action@v2
+              with:
+                files: coverage/cobertura-coverage.xml
```

Comment:
nit: should we add a newline to the end of file?

Reply:
New line added